src/types/any.rs

```diff
@@ -118,6 +118,39 @@ pub trait PyAnyMethods<'py>: crate::sealed::Sealed {
     where
         N: IntoPyObject<'py, Target = PyString>;
 
+    /// Retrieves an attribute value optionally.
+    ///
+    /// This is equivalent to the Python expression `getattr(self, attr_name, None)`, but uses
+    /// the Python 3.13+ `PyObject_GetOptionalAttr` API for efficiency. Unlike `getattr`, it
+    /// returns `None` if the attribute is not found instead of raising `AttributeError`.
```

Comment:
I would be downplay the efficiency gain, as many classes implementing `__getattr__` will necessarily still be raising `AttributeError` which the API is swallowing internally.

```suggestion
    /// This is equivalent to the Python expression `getattr(self, attr_name, None)`, which may
    /// be more efficient in some cases by simply returning `None` if the attribute is not found
    /// instead of raising `AttributeError`.
```

Reply:
Thanks & adopted your suggestions.